channeld/channeld.c

```diff
@@ -163,8 +163,9 @@ struct peer {
 
 	/* Empty commitments.  Spec violation, but a minor one. */
 	u64 last_empty_commitment;
-	/* It means if we are in reconnection process*/
-	bool reconnected;
+
+	/* It means if we have stored announcement signatures so far*/
+	bool remote_ann_stored;
```

Comment:
Like mentioned before we don't need to track this. If the remote end wants to give us a new (valid) signature we can certainly just overwrite any existing one.

Reply:
Yes, I simplied it in the new commit :)